src/components/app-configure/AppKeyspace.vue

```diff
@@ -141,33 +137,6 @@ function handleCancel() {
                     >
                       *Recommended
```

Comment:
Remove this block, we don't recommend it anymore

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/431/commits/2cfcff59411aab2d20a21eed12b045baf10e250e)